README.md

```diff
@@ -226,8 +226,8 @@ describe("Ignored specs", () -> {
 ```
 #### Ignoring Specs Spectrum style
 
-Spectrum allows you to specify pre-conditions on a block. These preconditions can include tagging.
-The `with(PreConditions,Block)` function is used to annotate a block with the preconditions. As ignoring is a common case, there is also an `ignore` function you can
+Spectrum allows you to specify preconditions on a block. These preconditions can include tagging.
+The `with(Preconditions,Block)` function is used to annotate a block with the preconditions. As ignoring is a common case, there is also an `ignore` function you can
```

Comment:
Replaced all occurrences of the following variant spellings with `precondition` consistently.

* preCondition
* pre condition
* pre-condition